src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/SourceBuiltArtifactsTests.cs

```diff
@@ -47,7 +51,7 @@ public void VerifyVersionFile()
             string sdkVersion = versionLines[1];
 
             // Find the expected SDK version by getting it from the SDK tarball
-            ExtractFileFromTarball(Config.SdkTarballPath, "./sdk/*/.version", outputDir);
+            Utilities.ExtractFileFromTarball(Config.SdkTarballPath, "./sdk/*/.version", outputDir);
```

Comment:
This change is required because Alpine doesn't support the `--wildcards` option like the other distros.